cordova-plugin-outline/apple/src/OutlinePlugin.swift

```diff
@@ -184,6 +184,15 @@ class OutlinePlugin: CDVPlugin {
     }
     event.message = "\(OutlinePlugin.kPlatform) report (\(uuid))"
 
+    // Remove device identifier. Note that we cannot use the beforeSerializeEvent callback,
+    // since contexts are only added after serialization when not present.
+    let serializedEvent = event.serialize()
+    var contexts = serializedEvent["contexts"] as? [String: [String: Any]]
+    contexts?["app"]?["device_app_hash"] = ""
+    event.context = Context()
+    // Setting the sanitized app context will prevent it from being added on serialization.
+    event.context?.appContext = contexts?["app"];
```

Comment:
Nit: Do we need the semicolon?

Reply:
Removed - context switch from TS 😅

Reply:
I had the same confusion, I kept looking around for `as?` on the Typescript docs thinking "I've never seen this before" before realizing this was Swift 😂